### PR TITLE
Add Object Version count histogram

### PR DESCRIPTION
**Pull request by klauspost**
## Description

Adds log10 histograms of object version count.

Requires: https://github.com/minio/madmin-go/pull/179

## How to test this PR?

Run and observe usage data.

## Types of changes
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-pools.sh | :heavy_check_mark: |
| mint-erasure.sh | :x: [more...](#47656622-0874-421e-afbc-f8fced8ae8da) |
| mint-compress-encrypt-dist-erasure.sh | :x: [more...](#afc2f764-b25b-4ff0-84a3-43d410ec3495) |
<a name="47656622-0874-421e-afbc-f8fced8ae8da"></a>
### 16739-a9e398b/mint-erasure.sh.log:
```
Running with
SERVER_ENDPOINT:      15.15.15.8:32708
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0
RUN_ON_FAIL:          0

To get logs, run 'docker cp 7a6e3000191a:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 7 seconds
(2/14) Running aws-sdk-java tests ... done in 2 seconds
(3/14) Running aws-sdk-php tests ... done in 42 seconds
(4/14) Running aws-sdk-ruby tests ... done in 8 seconds
(5/14) Running awscli tests ... done in 1 minutes and 14 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 15 seconds
(8/14) Running minio-go tests ... done in 34 seconds
(9/14) Running minio-java tests ... done in 18 seconds
(10/14) Running minio-js tests ... FAILED in 2 minutes and 23 seconds
{
  "name": "minio-js",
  "function": "\"after all\" hook in \"functional tests\"",
  "duration": 129207,
  "status": "FAIL",
  "error": "Error: connect ETIMEDOUT 15.15.15.8:32708 at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1159:16)"
}
(10/14) Running minio-py tests ... done in 1 minutes and 38 seconds
(11/14) Running s3cmd tests ... done in 15 seconds
(12/14) Running s3select tests ... done in 2 seconds
(13/14) Running versioning tests ... done in 3 minutes and 25 seconds

Executed 13 out of 14 tests successfully.
```
<a name="afc2f764-b25b-4ff0-84a3-43d410ec3495"></a>
### 16739-a9e398b/mint-compress-encrypt-dist-erasure.sh.log:
```
Running with
SERVER_ENDPOINT:      15.15.15.7:30985
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0
RUN_ON_FAIL:          0

To get logs, run 'docker cp 8cbd3e964d6e:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 8 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 43 seconds
(4/14) Running aws-sdk-ruby tests ... done in 9 seconds
(5/14) Running awscli tests ... done in 1 minutes and 16 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 18 seconds
(8/14) Running minio-go tests ... done in 47 seconds
(9/14) Running minio-java tests ... FAILED in 2 minutes and 11 seconds
{
  "name": "minio-java",
  "function": "makeBucket()",
  "args": "[basic check]",
  "duration": 129513,
  "status": "FAIL",
  "error": "java.net.ConnectException: Failed to connect to /15.15.15.7:30985 >>> [okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.kt:297), okhttp3.internal.connection.RealConnection.connect(RealConnection.kt:207), okhttp3.internal.connection.ExchangeFinder.findConnection(ExchangeFinder.kt:226), okhttp3.internal.connection.ExchangeFinder.findHealthyConnection(ExchangeFinder.kt:106), okhttp3.internal.connection.ExchangeFinder.find(ExchangeFinder.kt:74), okhttp3.internal.connection.RealCall.initExchange$okhttp(RealCall.kt:255), okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.kt:32), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109), okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.kt:95), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109), okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.kt:83), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109), okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.kt:76), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109), okhttp3.internal.connection.RealCall.getResponseWithInterceptorChain$okhttp(RealCall.kt:201), okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:517), java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128), java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628), java.base/java.lang.Thread.run(Thread.java:829)]"
}
(9/14) Running minio-js tests ... done in 51 seconds
(10/14) Running minio-py tests ... done in 3 minutes and 54 seconds
(11/14) Running s3cmd tests ... done in 17 seconds
(12/14) Running s3select tests ... done in 4 seconds
(13/14) Running versioning tests ... done in 10 minutes and 39 seconds

Executed 13 out of 14 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-erasure.sh | :x: [more...](#29f00eea-0dd7-489a-aaa8-412507f40d64) |
| mint-compress-encrypt-dist-erasure.sh | :x: [more...](#d0ea8073-bec3-4e97-948f-a22af4ddb83c) |
| mint-pools.sh | :x: [more...](#e5c688ba-9438-448e-bd87-ab0f847681ae) |
<a name="29f00eea-0dd7-489a-aaa8-412507f40d64"></a>
### 16739-b50542b/mint-erasure.sh.log:
```
Running with
SERVER_ENDPOINT:      15.15.15.5:32658
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0
RUN_ON_FAIL:          0

To get logs, run 'docker cp c45db924ec0a:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 8 seconds
(2/14) Running aws-sdk-java tests ... done in 2 seconds
(3/14) Running aws-sdk-php tests ... done in 41 seconds
(4/14) Running aws-sdk-ruby tests ... done in 8 seconds
(5/14) Running awscli tests ... done in 1 minutes and 14 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 15 seconds
(8/14) Running minio-go tests ... done in 38 seconds
(9/14) Running minio-java tests ... done in 18 seconds
(10/14) Running minio-js tests ... FAILED in 1 minutes and 20 seconds
{
  "name": "minio-js",
  "function": "removeObjectTagging on an object",
  "args": "bucketName:minio-js-test-tags-215fe2c3-0194-4fce-9aa7-82844baeec7e, objectName:datafile-100-kB,",
  "duration": 8,
  "status": "FAIL",
  "error": "Error: done() called multiple times in test <functional tests Object Tags API set, get and remove Tags on an object removeObjectTagging on an object_bucketName:minio-js-test-tags-215fe2c3-0194-4fce-9aa7-82844baeec7e, objectName:datafile-100-kB,> of file /mint/run/core/minio-js/test/functional-tests.js; in addition, done() received error: Error: read ETIMEDOUT at TCP.onStreamRead (internal/stream_base_commons.js:209:20) { errno: -110, code: 'ETIMEDOUT', syscall: 'read' } at /mint/run/core/minio-js/node_modules/mocha-steps/lib/step.js:64:11 at /mint/run/core/minio-js/test/functional-tests.js:1910:27 at ClientRequest.<anonymous> (node_modules/minio/dist/main/minio.js:574:9) at Socket.socketErrorListener (_http_client.js:475:9) at emitErrorNT (internal/streams/destroy.js:106:8) at emitErrorCloseNT (internal/streams/destroy.js:74:3) at processTicksAndRejections (internal/process/task_queues.js:82:21)"
}
(10/14) Running minio-py tests ... done in 1 minutes and 30 seconds
(11/14) Running s3cmd tests ... done in 16 seconds
(12/14) Running s3select tests ... done in 3 seconds
(13/14) Running versioning tests ... done in 3 minutes and 2 seconds

Executed 13 out of 14 tests successfully.
```
<a name="e5c688ba-9438-448e-bd87-ab0f847681ae"></a>
### 16739-b50542b/mint-pools.sh.log:
```
Running with
SERVER_ENDPOINT:      15.15.15.4:32316
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0
RUN_ON_FAIL:          0

To get logs, run 'docker cp 0f4224a5e8ef:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 7 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 43 seconds
(4/14) Running aws-sdk-ruby tests ... done in 8 seconds
(5/14) Running awscli tests ... done in 1 minutes and 16 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 18 seconds
(8/14) Running minio-go tests ... done in 59 seconds
(9/14) Running minio-java tests ... FAILED in 1 minutes and 29 seconds
{
  "name": "minio-java",
  "function": "setObjectTags()",
  "duration": 26,
  "status": "FAIL",
  "error": "java.io.IOException: unexpected end of stream on http://15.15.15.4:32316/... >>> [okhttp3.internal.http1.Http1ExchangeCodec.readResponseHeaders(Http1ExchangeCodec.kt:202), okhttp3.internal.connection.Exchange.readResponseHeaders(Exchange.kt:106), okhttp3.internal.http.CallServerInterceptor.intercept(CallServerInterceptor.kt:79), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109), okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.kt:34), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109), okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.kt:95), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109), okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.kt:83), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109), okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.kt:76), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109), okhttp3.internal.connection.RealCall.getResponseWithInterceptorChain$okhttp(RealCall.kt:201), okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:517), java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128), java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628), java.base/java.lang.Thread.run(Thread.java:829)]"
}
(9/14) Running minio-js tests ... done in 52 seconds
(10/14) Running minio-py tests ... done in 2 minutes and 19 seconds
(11/14) Running s3cmd tests ... done in 18 seconds
(12/14) Running s3select tests ... done in 4 seconds
(13/14) Running versioning tests ... done in 3 minutes and 6 seconds

Executed 13 out of 14 tests successfully.
```
<a name="d0ea8073-bec3-4e97-948f-a22af4ddb83c"></a>
### 16739-b50542b/mint-compress-encrypt-dist-erasure.sh.log:
```
Running with
SERVER_ENDPOINT:      15.15.15.6:32606
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0
RUN_ON_FAIL:          0

To get logs, run 'docker cp bca88d35a907:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 6 seconds
(2/14) Running aws-sdk-java tests ... done in 2 seconds
(3/14) Running aws-sdk-php tests ... done in 42 seconds
(4/14) Running aws-sdk-ruby tests ... done in 9 seconds
(5/14) Running awscli tests ... done in 1 minutes and 16 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 17 seconds
(8/14) Running minio-go tests ... done in 53 seconds
(9/14) Running minio-java tests ... done in 30 seconds
(10/14) Running minio-js tests ... FAILED in 30 minutes and 18 seconds
{
  "name": "minio-js",
  "function": "\"after all\" hook in \"functional tests\"",
  "duration": 7114,
  "status": "FAIL",
  "error": "S3Error: The bucket you tried to delete is not empty at Object.parseError (node_modules/minio/dist/main/xml-parsers.js:71:11) at /mint/run/core/minio-js/node_modules/minio/dist/main/transformers.js:166:22 at DestroyableTransform._flush (node_modules/minio/dist/main/transformers.js:90:10) at DestroyableTransform.prefinish (node_modules/readable-stream/lib/_stream_transform.js:129:10) at prefinish (node_modules/readable-stream/lib/_stream_writable.js:611:14) at finishMaybe (node_modules/readable-stream/lib/_stream_writable.js:620:5) at endWritable (node_modules/readable-stream/lib/_stream_writable.js:643:3) at DestroyableTransform.Writable.end (node_modules/readable-stream/lib/_stream_writable.js:571:22) at IncomingMessage.onend (internal/streams/readable.js:670:10) at endReadableNT (internal/streams/readable.js:1333:12) at processTicksAndRejections (internal/process/task_queues.js:82:21)"
}
(10/14) Running minio-py tests ... done in 2 minutes and 4 seconds
(11/14) Running s3cmd tests ... done in 17 seconds
(12/14) Running s3select tests ... done in 4 seconds
(13/14) Running versioning tests ... done in 3 minutes and 11 seconds

Executed 13 out of 14 tests successfully.
```
Deleting image on docker hub
Deleting image locally

File: cmd/data-usage-cache.go [hunk 1]
```diff
@@ -47,13 +47,17 @@ type dataUsageHash string
 // sizeHistogram is a size histogram.
 type sizeHistogram [dataUsageBucketLen]uint64
 
+// sizeHistogram is a size histogram.
```
Comment (krisis) [inline cmd/data-usage-cache.go]:
```suggestion
// versionsHistogram is a histogram of number of versions in an object.
```

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-erasure.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
| mint-pools.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-erasure.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
| mint-pools.sh | :x: [more...](#e18a08c4-903e-48bb-9ec3-1c7f7288f8b5) |
<a name="e18a08c4-903e-48bb-9ec3-1c7f7288f8b5"></a>
### 16739-fdcf997/mint-pools.sh.log:
```
Running with
SERVER_ENDPOINT:      15.15.15.3:32600
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0
RUN_ON_FAIL:          0

To get logs, run 'docker cp 4b867c576aab:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 8 seconds
(2/14) Running aws-sdk-java tests ... done in 2 seconds
(3/14) Running aws-sdk-php tests ... done in 43 seconds
(4/14) Running aws-sdk-ruby tests ... done in 9 seconds
(5/14) Running awscli tests ... done in 1 minutes and 16 seconds
(6/14) Running healthcheck tests ... done in 1 seconds
(7/14) Running mc tests ... done in 18 seconds
(8/14) Running minio-go tests ... done in 59 seconds
(9/14) Running minio-java tests ... done in 37 seconds
(10/14) Running minio-js tests ... done in 53 seconds
(11/14) Running minio-py tests ... FAILED in 2 minutes and 26 seconds
{
  "name": "minio-py:test_stat_object_version",
  "status": "FAIL",
  "args": {
    "bucket_name": "minio-py-test-458d203f-7e23-419b-8240-ce6a6a7bc728",
    "object_name": "4fe97f3a-e7d6-4c26-ae66-d1a8094f46f8-metadata",
    "length": 11534336,
    "data": "LimitedRandomReader(11 * MB)",
    "metadata": {
      "X-Amz-Meta-Testing": "value"
    },
    "content_type": "application/octet-stream"
  },
  "message": "('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))",
  "error": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.8/dist-packages/urllib3/connectionpool.py\", line 703, in urlopen\n    httplib_response = self._make_request(\n  File \"/usr/local/lib/python3.8/dist-packages/urllib3/connectionpool.py\", line 449, in _make_request\n    six.raise_from(e, None)\n  File \"<string>\", line 3, in raise_from\n  File \"/usr/local/lib/python3.8/dist-packages/urllib3/connectionpool.py\", line 444, in _make_request\n    httplib_response = conn.getresponse()\n  File \"/usr/lib/python3.8/http/client.py\", line 1348, in getresponse\n    response.begin()\n  File \"/usr/lib/python3.8/http/client.py\", line 316, in begin\n    version, status, reason = self._read_status()\n  File \"/usr/lib/python3.8/http/client.py\", line 277, in _read_status\n    line = str(self.fp.readline(_MAXLINE + 1), \"iso-8859-1\")\n  File \"/usr/lib/python3.8/socket.py\", line 669, in readinto\n    return self._sock.recv_into(b)\nConnectionResetError: [Errno 104] Connection reset by peer\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/mint/run/core/minio-py/tests.py\", line 126, in _call_test\n    func(log_entry, *args, **kwargs)\n  File \"/mint/run/core/minio-py/tests.py\", line 825, in test_stat_object_version\n    _test_stat_object(log_entry, sse, version_check=True)\n  File \"/mint/run/core/minio-py/tests.py\", line 794, in _test_stat_object\n    result = _CLIENT.put_object(\n  File \"/usr/local/lib/python3.8/dist-packages/minio/api.py\", line 1766, in put_object\n    raise exc\n  File \"/usr/local/lib/python3.8/dist-packages/minio/api.py\", line 1725, in put_object\n    upload_id = self._create_multipart_upload(\n  File \"/usr/local/lib/python3.8/dist-packages/minio/api.py\", line 1565, in _create_multipart_upload\n    response = self._execute(\n  File \"/usr/local/lib/python3.8/dist-packages/minio/api.py\", line 403, in _execute\n    return self._url_open(\n  File \"/usr/local/lib/python3.8/dist-packages/minio/api.py\", line 266, in _url_open\n    response = self._http.urlopen(\n  File \"/usr/local/lib/python3.8/dist-packages/urllib3/poolmanager.py\", line 376, in urlopen\n    response = conn.urlopen(method, u.request_uri, **kw)\n  File \"/usr/local/lib/python3.8/dist-packages/urllib3/connectionpool.py\", line 787, in urlopen\n    retries = retries.increment(\n  File \"/usr/local/lib/python3.8/dist-packages/urllib3/util/retry.py\", line 550, in increment\n    raise six.reraise(type(error), error, _stacktrace)\n  File \"/usr/local/lib/python3.8/dist-packages/urllib3/packages/six.py\", line 769, in reraise\n    raise value.with_traceback(tb)\n  File \"/usr/local/lib/python3.8/dist-packages/urllib3/connectionpool.py\", line 703, in urlopen\n    httplib_response = self._make_request(\n  File \"/usr/local/lib/python3.8/dist-packages/urllib3/connectionpool.py\", line 449, in _make_request\n    six.raise_from(e, None)\n  File \"<string>\", line 3, in raise_from\n  File \"/usr/local/lib/python3.8/dist-packages/urllib3/connectionpool.py\", line 444, in _make_request\n    httplib_response = conn.getresponse()\n  File \"/usr/lib/python3.8/http/client.py\", line 1348, in getresponse\n    response.begin()\n  File \"/usr/lib/python3.8/http/client.py\", line 316, in begin\n    version, status, reason = self._read_status()\n  File \"/usr/lib/python3.8/http/client.py\", line 277, in _read_status\n    line = str(self.fp.readline(_MAXLINE + 1), \"iso-8859-1\")\n  File \"/usr/lib/python3.8/socket.py\", line 669, in readinto\n    return self._sock.recv_into(b)\nurllib3.exceptions.ProtocolError: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))\n",
  "duration": 114404
}
(11/14) Running s3cmd tests ... done in 18 seconds
(12/14) Running s3select tests ... done in 5 seconds
(13/14) Running versioning tests ... done in 3 minutes and 6 seconds

Executed 13 out of 14 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-erasure.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
| mint-pools.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

**Review by krisis — APPROVED** [review]
LGTM. Minor comment on the comments.

**Review by harshavardhana — APPROVED** [review]

**Comment by harshavardhana** [comment]
@klauspost can you resolve the conflicts?

**Comment by harshavardhana** [comment]
> @klauspost can you resolve the conflicts?

@klauspost ^^

**Comment by klauspost** [comment]
@harshavardhana Done.

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
| mint-pools.sh | :heavy_check_mark: |
| mint-erasure.sh | :x: [more...](#7dcc94b9-a4b2-4e4e-a186-c2fcc884614a) |
<a name="7dcc94b9-a4b2-4e4e-a186-c2fcc884614a"></a>
### 16739-96e4005/mint-erasure.sh.log:
```
Running with
SERVER_ENDPOINT:      15.15.15.7:31031
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0
RUN_ON_FAIL:          0

To get logs, run 'docker cp ff49dbaffc4d:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 8 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 42 seconds
(4/14) Running aws-sdk-ruby tests ... done in 8 seconds
(5/14) Running awscli tests ... FAILED in 6 minutes and 13 seconds
{"name": "awscli", "duration": 318943, "function": "delete_bucket\n", "status": "FAIL", "error": "fatal error: Connect timeout on endpoint URL: "http://15.15.15.7:31031/awscli-mint-test-bucket-11905?list-type=2&prefix=&encoding-type=url"remove_bucket failed: Unable to delete all objects in the bucket, bucket will not be deleted."}
(5/14) Running healthcheck tests ... done in 0 seconds
(6/14) Running mc tests ... done in 15 seconds
(7/14) Running minio-go tests ... done in 34 seconds
(8/14) Running minio-java tests ... done in 18 seconds
(9/14) Running minio-js tests ... done in 46 seconds
(10/14) Running minio-py tests ... done in 1 minutes and 31 seconds
(11/14) Running s3cmd tests ... done in 17 seconds
(12/14) Running s3select tests ... done in 2 seconds
(13/14) Running versioning tests ... done in 3 minutes and 15 seconds

Executed 13 out of 14 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Pull request merged**